packages/xo-server/src/xo-mixins/jobs/index.js

```diff
@@ -154,6 +154,28 @@ export default class Jobs {
       xo.getLogger('jobs').then(logger => {
         this._logger = logger
       })
+
+      this._app.on('plugins:registered', () => {
+        ;this._jobs
+          .get()
+          .then(jobs =>
+            asyncMap(jobs, async job => {
+              if (job.runId === undefined) {
+                return
+              }
+
+              this._app.emit(
+                'job:terminated',
+                undefined,
+                job,
+                await this._app.getSchedule(job.scheduleId),
+                String(job.runId)
```

Comment:
Why cast?

Reply:
`job.runId` should be a string, the bug making it a number will be pushed to master soon.